lib/rubocop/cop/layout/space_inside_block_braces.rb

```diff
@@ -157,8 +162,17 @@ def multiline_block?(left_brace, right_brace)
           left_brace.first_line != right_brace.first_line
         end
 
-        def aligned_braces?(left_brace, right_brace)
-          left_brace.first_line == right_brace.last_column
+        def aligned_braces?(node, inner, right_brace)
+          column = expression_column(node)
```

Comment:
Can you update to the following and remove the new `expression_column` method and `child_send_node` matcher?

```suggestion
          column = column = node.loc.expression.column

```